app/controllers/addresses_controller.rb

```diff
@@ -0,0 +1,21 @@
+class AddressesController < ApplicationController
+  def edit
+    @address = Address.find params[:id]
+  end
+
+  def update
+    @address = Address.find(params[:id])
+    if @address.update(address_params)
+      flash[:notice] = 'Erfogreich geupdated'
+      redirect_to player_path @address.addressable
```

Comment:
this wont work, because address is a relation with user model. This will be changed soon to player, but now, it only works this way.